### PR TITLE
Update boletin-de-pediatria.csl

### PR DESCRIPTION
**Pull request by palantre**
A major upgrade, now based on Vancouver cls, for better formatting of web references.

### CODE REVIEW

File: boletin-de-pediatria.csl [hunk 1]
```diff
@@ -18,109 +18,140 @@
...
-            </group>
-          </if>
-          <else>
-            <group delimiter=" ">
-              <text variable="URL"/>
-              <text macro="accessed-date"/>
-            </group>
-          </else>
-        </choose>
+      <if variable="URL">
```
Comment (bwiernik) [inline boletin-de-pediatria.csl:63]: Please use logic like above to prefer DOI over URL and use a prefix "https://doi.org/"

File: boletin-de-pediatria.csl [hunk 2]
```diff
@@ -18,109 +18,140 @@
...
+          </choose>
+          <choose>
+            <if variable="URL">
+              <text term="internet" prefix="[" suffix="]" text-case="capitalize-first"/>
+            </if>
+          </choose>
+        </group>
+        <text macro="edition" prefix=" "/>
       </if>
+      <!--add event-name and event-place once they become available-->
```
Comment (bwiernik) [inline boletin-de-pediatria.csl:101]: "event" and "event-place" are available. Can you add these as necessary and remove the note?

**Review by bwiernik — CHANGES_REQUESTED** [review]
Thanks! A few comments

File: boletin-de-pediatria.csl (hunk 2 above)
Comment (palantre) [reply boletin-de-pediatria.csl:101]: I'll remove de note, as the are not necessary for this citation style.

File: boletin-de-pediatria.csl (hunk 1 above)
Comment (palantre) [reply boletin-de-pediatria.csl:63]: Thanks for the suggestion. I'll upload some changes in this macro.

Comment (palantre) [reply boletin-de-pediatria.csl:63]: I suggest:

```
  <macro name="access">
    <choose>
      <if variable="page" match="none">
        <choose>
	 <if variable="DOI" match="any">
            <group delimiter=". ">
              <text variable="DOI" prefix="DOI: https://doi.org/"/>
            </group>
          </if>
          <else>
            <group delimiter=" ">
	      <text term="available at" text-case="capitalize-first" suffix=": "/>
              <text variable="URL" />
            </group>
          </else>
        </choose>
	</if>
    </choose>
  </macro>
```

Comment (bwiernik) [reply boletin-de-pediatria.csl:63]: Use 

```
  <macro name="access">
    <choose>
      <if variable="page" match="none">
        <choose>
	 <if variable="DOI" match="any">
            <text variable="DOI" prefix="https://doi.org/"/>
          </if>
          <else>
            <group delimiter=" ">
	      <text term="available at" text-case="capitalize-first" suffix=": "/>
              <text variable="URL" />
            </group>
          </else>
        </choose>
	</if>
    </choose>
  </macro>
```

File: boletin-de-pediatria.csl (hunk 2 above)
Comment (bwiernik) [reply boletin-de-pediatria.csl:101]: Okay, please delete this comment from the style then

File: boletin-de-pediatria.csl (hunk 1 above)
Comment (palantre) [reply boletin-de-pediatria.csl]: OK

File: boletin-de-pediatria.csl (hunk 2 above)
Comment (palantre) [reply boletin-de-pediatria.csl]: OK. I've sent a new commit.

**Comment by bwiernik** [comment]
Thanks!

**Pull request merged**